### ISSUE TITLE
Unable to run project

### ISSUE THREAD

**Issue by testgithubd** [OfficeDev/office-ui-fabric-react#7442]
### Environment Information 
 
 
- __Package version(s)__: 
node: 10.14.1
npm: 6.6.0
- __Browser and OS versions__: (fill this out if relevant)
OS: Windows 10
#### Actual behavior:
 Unable to build the project. I did a git clone of the project and tried to build it but it's failing with an error ''C:\Office' is not recognized as an internal or external command,operable program or batch file.'

As suggested for Windows users, tried to use npm v5.6.0 but still unable to get it working.
 
#### Expected behavior:
 
A clean build and a runnable project.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) 
 
Requested priority: (Blocking, High, Normal, Low) High
 
Products/sites affected: (if applicable)

**Comment by micahgodbolt** [comment]
What commands did you use after cloning the project?

**Comment by testgithubd** [comment]
Followed the instructions in the Readme - 

1. git clone https://github.com/OfficeDev/office-ui-fabric-react.git
2. npm install For Windows developers, please read the next section before running this
3. npm start

**Comment by micahgodbolt** [comment]
at what part are you getting errors, the install or start? Also...just to make sure....did you cd into the fabric folder?

**Comment by testgithubd** [comment]
Yes :-), did cd into the project or there would have been no errors like posted. Error occurs in install stage itself. Also, as suggested for Windows, used an older version of npm to confirm, receiving the same error.

**Comment by micahgodbolt** [comment]
That warning about node 6 was before we started to support newer node version. Go ahead and try node 8.x. I've updated the read me

**Comment by testgithubd** [comment]
@micahgodbolt See above. Already using latest LTS v of node and npm. The error msg remains the same.

**Comment by micahgodbolt** [comment]
Drat, sorry for missing that. I just cloned/installed with node 8.12 without any issues. Trying node 10 to see if it works with latest LTS

**Comment by micahgodbolt** [comment]
Yup, i'm having a similar issue with node 10. Can you try 8.12?

**Comment by cliffkoh** [comment]
We should investigate the node 10 problem... Thanks for reporting.

**Comment by testgithubd** [comment]
@micahgodbolt  I tried using node 8.12 and getting the same error. Are you on Win 10 x64? 
What version of npm are u using?

Error: 'C:\Office' is not recognized as an internal or external command,
operable program or batch file.
Failed at the office-ui-fabric-react-repo@1.0.0 _rushInstall script.

**Comment by testgithubd** [comment]
Tried using rush install which did not throw error. But a consecutive rush build throws the following:
```
1 of 30: [@uifabric/prettier-rules] completed successfully in 0.04 seconds
2 of 30: [@uifabric/tslint-rules] completed successfully in 0.04 seconds

[@uifabric/set-version] started
module.js:550
    throw err;
    ^

Error: Cannot find module 'C:\Office'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
module.js:550
    throw err;
    ^

Error: Cannot find module 'C:\Office'
    at Function.Module._resolveFilename (module.js:548:15)
    at Function.Module._load (module.js:475:25)
    at Function.Module.runMain (module.js:694:10)
    at startup (bootstrap_node.js:204:16)
    at bootstrap_node.js:625:3
```

**Comment by micahgodbolt** [comment]
After updating npm (I had a REALLY old version) I was able to successfully install 

![image](https://user-images.githubusercontent.com/1434956/50254501-13e77800-03a3-11e9-8059-4a7ce1c63348.png)

**Comment by micahgodbolt** [comment]
is c:\Office where you are installing from? seems resolveFilename thinks that folder is a module to install.

**Comment by testgithubd** [comment]
Just did a fresh pull and install, issue remains the same. Although I am installing it to a sub dir in C: (it's a new machine), it's not C:\Office (there is no such folder).

![image](https://user-images.githubusercontent.com/45836381/50256133-b86cb880-03a9-11e9-8b82-69b95569aa36.png)

**Comment by testgithubd** [comment]
Tried installing to a new folder without 'Office' in the name and it works! This stuff needs documentation.

**Comment by testgithubd** [comment]
My intention of installing this project was to play around with the experimental components, but when I run the \office-ui-fabric-react\packages\experiments project, I get numerous errors such as:

```
1. Module not found: Error: Can't resolve '@uifabric/file-type-icons' in 'C:\test\office-ui-fabric-react\packages\experiments\src\components\FileTypeIcon\examples'
2. ERROR in C:/test/office-ui-fabric-react/packages/experiments/src/components/Chiclet/examples/Chiclet.Basic.Example.tsx(5,32):
TS2307: Cannot find module './Chiclet.Basic.Example.scss'.
3. ERROR in C:/test/office-ui-fabric-react/packages/experiments/src/components/Tile/examples/Tile.Media.Example.tsx(8,42):
TS2307: Cannot find module './Tile.Example.scss'.
ERROR in C:/test/office-ui-fabric-react/packages/experiments/src/components/Tile/Tile.tsx(6,35):
TS2307: Cannot find module './Tile.scss'.
```

**Comment by cliffkoh** [comment]
Hey @testgithubd did you build the experiments folder yet? Assuming you have rush globally available, `rush build --to @uifabric/experiments`. After that, you should be able to do `NPM start` from within the experiments package.

Regarding the spacing issue, we frankly were not aware there was this limitation as you're the first person to report hitting this :) @ecraig12345 will investigate to see if this is something we can cheaply improve on our end.

**Comment by testgithubd** [comment]
@cliffkoh Thanks, I was using npm to build which didn't work. Using rush worked!

**Comment by ecraig12345** [comment]
Debugged this to try to figure out which part of the build system doesn't like spaces in paths... 

Definitely one issue is with the way `install-run-rush.js` spawns Rush in a child process, and I have a fix for that at https://github.com/Microsoft/web-build-tools/pull/1003. 

There's also some issue with either Fabric's build scripts or one of their dependencies--the failure to build `set-version` occurs during the Jest step, but I'm still working on figuring out why.

**Comment by micahgodbolt** [comment]
User was able to resolve the building issue by removing spaces from their path name. I will close this issue up as title/history aren't relevant. @ecraig12345 can you start up a new issue for that work?
